cohere/responses/custom_model.py

```diff
@@ -81,6 +81,9 @@ class ModelMetric(CohereObject):
     step_num: int
     loss: float
     accuracy: float
+    f1: float
```

Comment:
Are these optional?

Reply:
Looking at blob heart it doesn't seem like it:
https://github.com/cohere-ai/blobheart/blob/9debecc7a68d9311662d9b1e183ba09504e7bec6/proto/blobheart.gen.go#L1834-L1833

Reply:
It has omitempty for all these fields what happens when no field is returned?

Reply:
Is this possible though if the datatype is float32 and not *float32? Wouldn't it be 0 but not missing?

Reply:
Or wait does it drop the values if they are 0? Let me check that.

Reply:
>The "omitempty" option specifies that the field should be omitted from the encoding if the field has an empty value, defined as false, 0, a nil pointer, a nil interface value, and any empty array, slice, map, or string.

Okay my bad, though this was only true for nil.